lib/dialects/mssql/index.js

```diff
@@ -369,7 +361,14 @@ Object.assign(Client_MSSQL.prototype, {
       default:
         return response;
     }
-  },
+  }
+}
+
+// until node 12+
```

Comment:
what feature from Node 12 could be used here?

Reply:
class field properties

Reply:
oh, they are declared not on prototypes

Reply:
deleted this lines